Tasks/google-play-rollout-update/task.json

```diff
@@ -13,8 +13,8 @@
         "npm"
     ],
     "version": {
-        "Major": "1",
-        "Minor": "129",
+        "Major": "2",
+        "Minor": "135",
```

Comment:
The tasks don't have to be updated, it might be efficient to not update the task versions to avoid having to update them on the server when they didn't change.